src/event-actions/on-app-initialized.ts

```diff
@@ -1,12 +1,12 @@
-import {onCheckAvailability} from '@app/event-actions/on-check-availability';
 import {onRemoteConfigSync} from '@app/event-actions/on-remote-config-sync';
+import {EthRpcEndpointAvailability} from '@app/helpers/eth-rpc-endpoint-availability';
 import {RemoteConfig} from '@app/services/remote-config';
 
 /**
  * @description Called first when the app is initialized. Load remote config and check availability endpoints.
  */
 export async function onAppInitialized() {
+  EthRpcEndpointAvailability.checkEthRpcEndpointAvailability();
```

Comment:
In config can come new endpoints, for this reason check should be after fetching the config